main.tf

```
@@ -0,0 +1,115 @@
+terraform {
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 4.16"
+    }
+  }
+
+  required_version = ">= 1.2.0"
+}
+
+provider "aws" {
+  region = "us-east-2"
+}
+
+resource "aws_vpc" "Main" {
+  cidr_block       = "10.0.0.0/24"
+  instance_tenancy = "default"
+  tags = {
+    name = "Main"
+  }
+}
+resource "aws_internet_gateway" "IGW" {
+  vpc_id = aws_vpc.Main.id
+}
+resource "aws_subnet" "public_A" {
+  vpc_id     = aws_vpc.Main.id
+  cidr_block = "10.0.0.0/26"
+  map_public_ip_on_launch = true
+  tags = {
+    name = "public_A"
+  }
+}
+resource "aws_subnet" "public_B" {
+  vpc_id     = aws_vpc.Main.id
+  cidr_block = "10.0.0.64/26"
+  map_public_ip_on_launch = true
+  tags = {
+    name = "public_B"
+  }
+}
```

Please separate separet eeach top-level resource using a newline.
```suggestion
resource "aws_internet_gateway" "IGW" {
  vpc_id = aws_vpc.Main.id
}

resource "aws_subnet" "public_A" {
  vpc_id     = aws_vpc.Main.id
  cidr_block = "10.0.0.0/26"
  map_public_ip_on_launch = true
  tags = {
    name = "public_A"
  }
}

resource "aws_subnet" "public_B" {
  vpc_id     = aws_vpc.Main.id
  cidr_block = "10.0.0.64/26"
  map_public_ip_on_launch = true
  tags = {
    name = "public_B"
  }
}
```
Please fix this here and in all other similar places in the code